coriolis/osmorphing/osmount/base.py

```diff
@@ -107,12 +107,27 @@ def _get_pvs(self):
             if line == "":
                 continue
             line = line.strip().split(":")
-            if pvs.get(line[1]) is None:
-                pvs[line[1]] = [line[0], ]
+            if len(line) >= 2:
+                if pvs.get(line[1]) is None:
+                    pvs[line[1]] = [line[0], ]
+                else:
+                    pvs[line[1]].append(line[0])
             else:
-                pvs[line[1]].append(line[0])
+                LOG.warn(
+                    "Ignoring improper `pvdisplay` output entry: %s" % line)
```

Comment:
NIT: `LOG.bla` automatically formats so the `%` wasn't necessary.
It doesn't really matter though so I'll be merging it anyway. Thanks!